styles.css

```diff
@@ -533,3 +533,117 @@ button[enabled] {
   border-radius: 150px;
   margin-right: 12px;
 }
+
+.contact-form,
+.my-form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 12px;
+  background-color: #e5e5e5;
+}
+
+.contact-form-content {
+  margin-top: 108px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 70px;
+}
```

Comment:
- [ ] Kindly align your paragraph correctly in order to look exactly like the provided design. :wink: 

   - ### Your design
     ![image](https://user-images.githubusercontent.com/87197412/186628622-2c029b77-edf2-451c-a709-7e7ca9d18cbb.png)


  - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186628462-f93f96a4-c87f-4392-98c8-9745e16d2a1c.png)